dataeng/resources/tableau-backup.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+set -ex
+
+
+ssh -tt centos@$TABLEAU_SERVER_IP -o StrictHostKeyChecking=no "
+
+sudo -H -u root bash -c 'aws s3 cp /var/opt/tableau/tableau_server/data/tabsvc/files/backups/$(date '+%Y.%m.%d').11.00-tableau-data-backup.tsbak s3://edx-analytics-tableau/backups/$(date '+%Y.%m.%d').11.00-tableau-data-backup.tsbak'
+sudo -H -u root bash -c 'rm /var/opt/tableau/tableau_server/data/tabsvc/files/backups/$(date '+%Y.%m.%d').11.00-tableau-data-backup.tsbak'
```

Comment:
Is there a reason we need to run this as root? I usually think it should be avoided wherever possible. Maybe we can look into changing the ownership of the directories or files in play here.

Reply:
I have tried multiple ways but nothing else work. I feel running these two commands as root is less risky than than changing directory ownership. It will require as to create a new user group and add all the required users into that group and then change owership of tableau specific directory so that's why I was hesitant doing that and opted this way.

Reply:
So I have now changed the default location for backup files to get stored. I created a new folder '/home/tableau_backups' and gave 'centos' and 'tableau' users access to that folder

Reply:
In this way this is pretty restrictive for `centos` as it was before and we can avoid running these commands as root. As we will now be only iterating with  `/home/tableau_backups `